tests/integration/targets/inventory_lxd/tasks/lxd.yml

```diff
@@ -0,0 +1,39 @@
+---
+plugin: lxd
```

Comment:
This is **not an Ansible tasks file**. Remove everything from tests/integration/ that is not an integration test!

Reply:
This is an example of a config file for the inventory. Is there a other place to ship it to the users? If not, i will delete it.

Reply:
It is used by the unit tests, so it belongs into `tests/unit/...`, not in `tests/integration/...`.

Reply:
done